input/fsh/au-erequesting-servicerequest.fsh

```diff
@@ -2,15 +2,12 @@ Profile: AUeRequestingServiceRequest
 Parent: AUBaseDiagnosticRequest
 Id: au-erequesting-servicerequest
 Title: "AU eRequesting ServiceRequest"
-Description: "This profile defines a service request structure to represent a request for a diagnostic investigation. It is based on the [AU Base Diagnostic Request](http://hl7.org.au/fhir/StructureDefinition/au-diagnosticrequest)."
+Description: "This profile sets minimum expectations for a ServiceRequest resource that is used to record, search, and fetch information about diagnostic requests, plans, or orders for a service for a patient. It is based on the [AU Base Diagnostic Request](https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-diagnosticrequest.html) and identifies the additional constraints, extensions, vocabularies and value sets that **SHALL** be present in the ServiceRequest when conforming to this profile."
```

Comment:
From the description remove the text ', plans or orders for a service for'.

Reply:
Done